app/src/main/java/com/example/concordia_campus_guide/Models/Coordinates.java

```diff
@@ -22,6 +22,11 @@ public boolean equals(Object o) {
                 longitude == that.longitude;
     }
 
+    @Override
+    public int hashCode() {
```

Comment:
why do we need this?

Reply:
It was suggested by sonar lint. Basically the reason is that if we are overwriting the equals method, we should also overwrite the hashcode method. This is mostly useful when we use hashmaps.